src/main/java/notification_window/Warn.java

```diff
@@ -0,0 +1,45 @@
+package notification_window;
+
+import drink.Drink;
+import helpers.Deserializer;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import java.util.HashMap;
+
+public class Warn {
+    protected static ArrayList<Drink> drinks = new ArrayList<>();
+    private static final Deserializer searchDeserializer = new Deserializer();
+    public static void constructDrinkDataBase() throws IOException, ClassNotFoundException {
+        searchDeserializer.deserialize("./data/drinks");
+        drinks = (ArrayList<Drink>) searchDeserializer.getObject();
+    }
+
+
+    public static String warnwhat(HashMap<Drink, String> itemList){
+        ArrayList<Drink> cart_items = new ArrayList<>();
+        ArrayList<String> dang = new ArrayList<>();
+        dang.add("egg");
+        dang.add("milk");
```

Comment:
Overall good job, but I recommend changing the variable name to a more understanding one.